app/(tabs)/expenses/balance.tsx

```diff
@@ -3,33 +3,61 @@ import { Text, View, } from '../../../components/Themed';
 import { FlatList } from 'react-native-gesture-handler';
 import { calculateBalance } from '../../../helpers/calculateBalance';
 import Colors from '../../../constants/Colors';
+import { Expense} from '../../../helpers/calculateExpenses';
+import { expensesHandle } from '../../../handlers/expenses';
+import { useEffect, useRef, useState } from 'react';
 
+function expenseListCmp(cmp1 : Expense[], cmp2 : Expense[]){
```

Comment:
What cmp stand for? Can you find better name, seems like an equals method based on the content